.gitignore

```diff
@@ -0,0 +1 @@
+/vendor/
```

Comment:
Solution is to publish a composer.lock or ignore it. I added a comment under this one to explain myself better.